lib/linter.coffee

```diff
@@ -156,6 +156,34 @@ class Linter
 
     process = new BufferedProcess({command, args, options,
                                   stdout, stderr, exit})
+    process.onWillThrowError (err) =>
+      return unless err?
+      if err.error.code is 'ENOENT'
+        ignored = atom.config.get('linter.ignoredLinterErrors')
+        subtle = atom.config.get('linter.subtleLinterErrors')
+        if @linterName in subtle
+          # Show a small notification at the bottom of the screen
+          new MessagePanelView(
+            title: "#{@linterName} linter not installed"
+          ).attach()
```

Comment:
I would prefere:

```
message = new MessagePanelView
  title: "#{@linterName} linter not installed"
message.attach()
```

Invoking construction function right after it is a bad habit, see: http://stackoverflow.com/a/3686284/4177885